fermenter-mda-test-utils/src/main/java/org/technologybrewery/fermenter/mda/test/utils/TestUtil.java

```diff
@@ -0,0 +1,82 @@
+package org.technologybrewery.fermenter.mda.test.utils;
+
+import org.technologybrewery.fermenter.mda.GenerateSourcesHelper;
+import org.technologybrewery.fermenter.mda.element.ExpandedProfile;
+import org.technologybrewery.fermenter.mda.element.Target;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TestUtil {
```

Comment:
S: Let's add Javadocs since these are public APIs.  Especially at the class level.